docs: change variable naming in `blas/base/caxpy`

ShabiShett07 opened the pull request:
---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes. report:
  - task: lint_filenames status: passed
  - task: lint_editorconfig status: passed
  - task: lint_markdown status: passed
  - task: lint_package_json status: na
  - task: lint_repl_help status: passed
  - task: lint_javascript_src status: passed
  - task: lint_javascript_cli status: na
  - task: lint_javascript_examples status: passed
  - task: lint_javascript_tests status: passed
  - task: lint_javascript_benchmarks status: passed
  - task: lint_python status: na
  - task: lint_r status: na
  - task: lint_c_src status: missing_dependencies
  - task: lint_c_examples status: missing_dependencies
  - task: lint_c_benchmarks status: missing_dependencies
  - task: lint_c_tests_fixtures status: na
  - task: lint_shell status: na
  - task: lint_typescript_declarations status: passed
  - task: lint_typescript_tests status: passed
  - task: lint_license_headers status: passed ---

none

## Description

> What is the purpose of this pull request?

This pull request:

-  changes the variable names to align with current conventions.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   none

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

ShabiShett07 committed:
docs: change variable naming in blas/base/zdscal

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: passed
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: passed
  - task: lint_javascript_tests
    status: passed
  - task: lint_javascript_benchmarks
    status: passed
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: missing_dependencies
  - task: lint_c_examples
    status: missing_dependencies
  - task: lint_c_benchmarks
    status: missing_dependencies
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: passed
  - task: lint_typescript_tests
    status: passed
  - task: lint_license_headers
    status: passed
---

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
````diff
@@ -205,51 +205,51 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
 ```
 
-#### c_caxpy( N, ca, \*CX, strideX, \*CY, strideY )
+#### c_caxpy( N, alpha, \*CX, strideX, \*CY, strideY )
````
@ShabiShett07 You should also change `CX` to `X` and `CY` to Y`. Applies here and elsewhere.

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
```diff
@@ -205,51 +205,51 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
+#### c_caxpy( N, alpha, \*CX, strideX, \*CY, strideY )
 
-Scales values from `cx` by `ca` and adds the result to `cy`.
+Scales values from `x` by `alpha` and adds the result to `y`.
```
```suggestion
Scales values from `X` by `alpha` and adds the result to `Y`.
```

This should match the parameter names above.

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
```diff
@@ -205,51 +205,51 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
 -   **N**: `[in] CBLAS_INT` number of indexed elements.
--   **ca**: `[in] stdlib_complex64_t` scalar constant.
+-   **alpha**: `[in] stdlib_complex64_t` scalar constant.
 -   **CX**: `[in] void*` input array.
```
```suggestion
-   **X**: `[in] void*` input array.
```

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
```diff
@@ -205,51 +205,51 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
+-   **alpha**: `[in] stdlib_complex64_t` scalar constant.
 -   **CX**: `[in] void*` input array.
 -   **strideX**: `[in] CBLAS_INT` index increment for `CX`.
 -   **CY**: `[inout] void*` output array.
```
```suggestion
-   **Y**: `[inout] void*` output array.
```

And also the descriptions above and below.

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
````diff
@@ -205,51 +205,51 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
 
 ```c
-void c_caxpy( const CBLAS_INT N, const stdlib_complex64_t ca, const void *CX, const CBLAS_INT strideX, void *CY, const CBLAS_INT strideY );
+void c_caxpy( const CBLAS_INT N, const stdlib_complex64_t alpha, const void *CX, const CBLAS_INT strideX, void *CY, const CBLAS_INT strideY );
````
Same thing here and below.

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/test/test.caxpy.native.js:
```diff
@@ -275,37 +275,37 @@ tape( 'the function supports a `cy` stride', opts, function test( t ) {
 	t.end();
 });
 
 tape( 'if provided `ca` parameter equal to `0`, the function returns the second input array unchanged', opts, function test( t ) {
```
```suggestion
tape( 'if provided `alpha` parameter equal to `0`, the function returns the second input array unchanged', opts, function test( t ) {
```

kgryte committed:
test: update description

Signed-off-by: Athan <kgryte@gmail.com>

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/test/test.caxpy.native.js:
```diff
@@ -80,14 +80,14 @@ tape( 'the function has an arity of 6', opts, function test( t ) {
+tape( 'the function scales elements from `x` by `ca` and adds the result to `y`', opts, function test( t ) {
 	var expected;
 	var viewY;
 	var ca;
```
In the other files, you changed `ca` to `alpha`. Do you want to do the same here and elsewhere in this file?

kgryte reviewed (changes requested):
Apart from the README and the one test file, this looks to be in decent shape.

ShabiShett07 committed:
chore: update variables

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: passed
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

ShabiShett07 commented:
@kgryte, sorry for having errors again.

ShabiShett07 committed:
chore: update variables

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: na
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
```diff
@@ -205,60 +205,60 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
+#### c_caxpy( N, alpha, \*X, strideX, \*Y, strideY )
 
-Scales values from `cx` by `ca` and adds the result to `cy`.
+Scales values from `X` by `alpha` and adds the result to `y`.
```
```suggestion
Scales values from `X` by `alpha` and adds the result to `Y`.
```

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
```diff
@@ -205,60 +205,60 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
-float cy[] = { -1.0f, -2.0f, -3.0f, -4.0f, -5.0f, -6.0f, -7.0f, -8.0f };
-const stdlib_complex64_t ca = stdlib_complex64( 2.0f, 2.0f );
+float X[] = { 1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f };
+float y[] = { -1.0f, -2.0f, -3.0f, -4.0f, -5.0f, -6.0f, -7.0f, -8.0f };
```
```suggestion
float Y[] = { -1.0f, -2.0f, -3.0f, -4.0f, -5.0f, -6.0f, -7.0f, -8.0f };
```

kgryte commented on lib/node_modules/@stdlib/blas/base/caxpy/README.md:
```diff
@@ -205,60 +205,60 @@ logEach( '(%s)*(%s) + (%s) = %s', ca, cx, cyc, cy );
+const stdlib_complex64_t alpha = stdlib_complex64( 2.0f, 2.0f );
 
-c_caxpy( 4, ca, (void *)cx, 1, (void *)cy, 1 );
+c_caxpy( 4, alpha, (void *)X, 1, (void *)y, 1 );
```
```suggestion
c_caxpy( 4, alpha, (void *)X, 1, (void *)Y, 1 );
```

kgryte committed:
Apply suggestions from code review

Signed-off-by: Athan <kgryte@gmail.com>

kgryte reviewed (approved):
LGTM
